client/api/smb/v1beta2/api.proto

```diff
@@ -2,8 +2,21 @@ syntax = "proto3";
 
 package v1beta2;
 
+import "google/protobuf/descriptor.proto";
+
 option go_package = "github.com/kubernetes-csi/csi-proxy/client/api/smb/v1beta2";
 
+extend google.protobuf.FieldOptions {
+    // Indicates that a field MAY contain information that is sensitive
+    // and MUST be treated as such (e.g. not logged).
+    bool csi_secret = 1059;
+
+    // Indicates that this field is OPTIONAL and part of an experimental
+    // API that may be deprecated and eventually removed between minor
+    // releases.
+    bool alpha_field = 1060;
```

Comment:
it looks like we aren't using alpha_field, only csi_secret

Reply:
In this `fieldOption` `bool aplha_field` is optional field  i don't have exact idea on this if this not impact changes, so should i remove this?

Reply:
I think that it's referring to the field that's going to be annotated, in this example from the CSI Spec it'd be the `volume_mount_group` field

```
  message MountVolume {
    // The filesystem type. This field is OPTIONAL.
    // An empty string is equal to an unspecified field value.
    string fs_type = 1;

    // The mount options that can be used for the volume. This field is
    // OPTIONAL. `mount_flags` MAY contain sensitive information.
    // Therefore, the CO and the Plugin MUST NOT leak this information
    // to untrusted entities. The total size of this repeated field
    // SHALL NOT exceed 4 KiB.
    repeated string mount_flags = 2;

    // If SP has VOLUME_MOUNT_GROUP node capability and CO provides
    // this field then SP MUST ensure that the volume_mount_group
    // parameter is passed as the group identifier to the underlying
    // operating system mount system call, with the understanding
    // that the set of available mount call parameters and/or
    // mount implementations may vary across operating systems.
    // Additionally, new file and/or directory entries written to
    // the underlying filesystem SHOULD be permission-labeled in such a
    // manner, unless otherwise modified by a workload, that they are
    // both readable and writable by said mount group identifier.
    // This is an OPTIONAL field.
    string volume_mount_group = 3 [(alpha_field) = true];
  }
```

Reply:
@mauriciopoppe Thanks for your input , i  will addressed this ASAP